cmd/identities/get.go

```diff
@@ -29,10 +37,23 @@ func NewGetCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := cliclient.NewClient(cmd)
 
+			// we check includeCreds argument is valid
+			for _, opt := range includeCreds {
+				e := stringsx.SwitchExact(opt)
+				if !e.AddCase("oidc") {
+					cmd.PrintErrln(`You have to put a valid value of credentials type to be included, try --help for details.`)
+					return nil
+				}
+			}
+
 			identities := make([]kratos.Identity, 0, len(args))
 			failed := make(map[string]error)
 			for _, id := range args {
-				identity, _, err := c.V0alpha2Api.AdminGetIdentity(cmd.Context(), id).Execute()
+				identity, _, err := c.V0alpha2Api.
+					AdminGetIdentity(cmd.Context(), id).
+					IncludeCredential(includeCreds).
```

Comment:
Will this also work if there is no credential? Not sure how the SDK works :sweat_smile:

Reply:
But I guess it works as the current test cases seem to pass.

Reply:
Well, as in specified in https://github.com/ory/kratos/blob/31a3c19034456dac1810d3c29c87ab49ad7cfd94/identity/identity.go#L324 if no OIDC, then it will process but just do nothing